torntools/manifest.json

```diff
@@ -214,6 +214,12 @@
 			"js": ["scripts/content/casino/ttBlackjack.js"],
 			"run_at": "document_end"
 		},
+		{
+			"matches": ["https://www.torn.com/loader.php?sid=attack*"],
+			"css": ["scripts/content/loader/ttLoader.css"],
+			"js": ["scripts/content/loader/ttLoader.js"],
```

Comment:
The file names should be changed, at it's specific for attacks.

Reply:
Done.